sycl/source/detail/graph_impl.cpp

```diff
@@ -311,6 +302,8 @@ graph_impl::add(sycl::detail::CG::CGTYPE CGType,
 
   const std::shared_ptr<node_impl> &NodeImpl =
       std::make_shared<node_impl>(CGType, std::move(CommandGroup));
+  MNodeStorage.push_back(NodeImpl);
+
   if (!Deps.empty()) {
     for (auto &N : Deps) {
       N->registerSuccessor(NodeImpl, N); // register successor
```

Comment:
Can we remove these comment lines since we're refactoring, they don't add any value